manifests/service.pp

```diff
@@ -6,7 +6,6 @@
     enable     => true,
     hasstatus  => true,
     hasrestart => true,
-    require    => [Service[$candlepin::db_type]],
```

Comment:
I think ```candlepin::database::postgresql``` does it explicitly and ```candlepin``` also has a chaining. For MySQL it doesn't appear to include the database server or manage the DB so that might not even work unless the user does that manually. That's why I'm pretty sure this is covered elsewhere.